_data/simple-icons.json

```diff
@@ -13794,6 +13794,16 @@
             "source": "https://www.streamlit.io/brand",
             "guidelines": "https://www.streamlit.io/brand"
         },
+        {
+            "title": "StreamRunners",
+            "hex": "6644F8",
+            "source": "https://streamrunners.fr/index",
+            "aliases": {
+                "aka": [
+                    "sr"
+                ]
+            }
```

Comment:
Could you provide a source for this abbreviation? Unless they refer to themselves officially as "sr" anywhere then we shouldn't include it as an alias.

Reply:
Regarding this, our community and the staff often use "sr" to avoid saying `StreamRunners`, which is longer. I don't have a real source regarding this (it's not documented in an officiel file/support), therefore I can remove the `alias` if the explanation is not enough to keep it!

Reply:
If it's just a community and internal shorthand rather than an official alternative name then it's best not to add it here.